src/test/js/notMultibranch/freestyleRunning.js

```diff
@@ -0,0 +1,27 @@
+/** @module freestyleRunning
+ * @description Check can run non multibranch
+ */
+module.exports = {
+    /** Create freestyle Job "hijo"*/
+    'Step 01': function (browser) {
+        const freestyleCreate = browser.page.freestyleCreate().navigate();
```

Comment:
We need to make sure that we use unique project names, if not running the whole suite may reviel other results then a single run that is not good :ant: